sonic-thermalctld/tests/test_thermalctld.py

```diff
@@ -180,7 +180,11 @@ def test_update_fan_with_exception(self):
         assert fan.get_status_led() == MockFan.STATUS_LED_COLOR_RED
         assert fan_updater.log_warning.call_count == 1
 
-        fan_updater.log_warning.assert_called_with("Failed to update fan status - Failed to get speed")
+        # TODO: Clean this up once we no longer need to support Python 2
+        if sys.version_info.major == 3:
```

Comment:
How about we add a helper function like below:
```python
def get_exception_message(msg):
    if sys.version_info.major == 3:
        return 'Exception(\'{},\')'.format(msg)
    else:
        return 'Exception(\'{}\')'.format(msg)
```

And we don't have to change everywhere when removing python2 support.

Reply:
Thank you for the suggestion. However, I believe this unit test code will not change much, if at all before we remove Python 2 support. And with things the way they are, all we have to do is delete lines. If we make the change you suggested, there will actually be a bit more work needed when removing Python 2 support. Therefore, I prefer we just leave things as-is for the time being.